src/test/java/org/primeframework/mvc/PrimeBaseTest.java

```diff
@@ -218,6 +218,7 @@ protected void configure() {
         bind(MessageObserver.class).toInstance(messageObserver);
         bind(MetricRegistry.class).toInstance(metricRegistry);
         bind(UserLoginSecurityContext.class).to(MockUserLoginSecurityContext.class);
+        bind(URLPrinter.class);
```

Comment:
This is ok - but it is not necessary to bind a concrete class. Guice knows how to call `new` when you ask to `@Inejct` a concrete class.

Reply:
I swore earlier it was not injecting this and I needed to add that (and then I probably got seduced by https://github.com/google/guice/wiki/UntargettedBindings). In reality, I think what was going on is the example test class I created did not inherit from a base class that wired up a Guice module (or use a Guice module/TestNG annotation of its own). Fixed.